treeherder/seta/job_priorities.py

```diff
@@ -67,7 +68,7 @@ def seta_job_scheduling(self, project, build_system_type, priority=None):
             job_priorities = []
             for jp in self._query_job_priorities(priority=priority,
                                                  excluded_build_system_type='buildbot'):
-                if jp.has_expired():
+                if jp.has_expired() or jp.expiration_date == THE_FUTURE:
```

Comment:
Read this, if the job has expired (the expiration date is in the past or has none) then add it OR if the job is not ever going to be expired (I doubt this code will be in execution by 2100).

This code was preventing entries from preseed to show up in the web API.